pkgs/by-name/dp/dprint/plugins/default.nix

```diff
@@ -2,6 +2,7 @@
   lib,
   fetchurl,
   stdenv,
+  dprint,
```

Comment:
```suggestion
  dprint,
  writableTmpDirAsHomeHook,
```

Reply:
As I understand it, writableTmpDirAsHomeHook cannot replace DPRINT_CACHE_DIR. It is not a HOME directory. Did it work on your environment?

Reply:
Ah, understood. Creating writable $HOME makes it possible to write $HOME/.cache :heavy_check_mark: